po/sk.po

```diff
@@ -4,16 +4,16 @@ msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2021-03-19 16:17+0100\n"
-"PO-Revision-Date: 2021-01-29 10:40+0000\n"
+"PO-Revision-Date: 2021-03-22 12:56+0000\n"
 "Last-Translator: Matej Marusak <mmarusak@redhat.com>\n"
-"Language-Team: Slovak <https://translate.fedoraproject.org/projects/cockpit/"
-"master/sk/>\n"
+"Language-Team: Slovak <https://translate.fedoraproject.org/projects/"
+"cockpit-machines/main/sk/>\n"
 "Language: sk\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"
-"X-Generator: Weblate 4.4.2\n"
+"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2\n"
```

Comment:
This looks suspicious -- @mmarusak, what was the hack again to deal with these pesky semicolons?

Reply:
The hack was to always remove them. So this PR is correct.
What is not correct is main - https://github.com/cockpit-project/cockpit-machines/blob/main/po/ko.po
I believe this one will break the page.
@KKoukiou how did you get translation files into main? I though we will get all of them on the first refresh, but it seems there are already some files.
I would suggest removing all of them in main and letting bots do their work.

Reply:
Ack, thanks! So I suppose this is good enough to land?

I think Katerina just copied the cockpit translations and locally ran `make update-po` to merge them.

Reply:
Right, lets land this and I'll send update with fixed plural-forms